docker-entrypoint.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# If long & short hostnames are not the same, use long hostnames
+if [ $(hostname) != $(hostname -s) ]; then
```

Comment:
Can you please quote both of these? ie, `if [ "$(hostname)" != "$(hostname -s)" ]; then`